pycbc/events/triggers.py

```diff
@@ -270,21 +273,14 @@ def get_found_param(injfile, bankfile, trigfile, param, ifo):
         return trigfile[ifo][param][:][foundtrg]
     else:
         b = bankfile
-        found_param_dict = {
-          "mtotal" : (b['mass1'][:] + b['mass2'][:])[foundtmp],
-          "mchirp" : conversions.mchirp_from_mass1_mass2(b['mass1'][:],
-                     b['mass2'][:])[foundtmp],
-          "eta"    : conversions.eta_from_mass1_mass2(b['mass1'][:],
-                     b['mass2'][:])[foundtmp],
-          "effective_spin" : conversions.chi_eff(b['mass1'][:],
-                                                 b['mass2'][:],
-                                                 b['spin1z'][:],
-                                                 b['spin2z'][:])[foundtmp]
-        }
-
-    return found_param_dict[param]
-
-def get_inj_param(injfile, param, ifo):
+        found_param = trstats.get_param(param, args, b['mass1'][:],
+                                        b['mass2'][:],
+                                        b['spin1z'][:],
+                                        b['spin2z'][:])[foundtmp]
+
+    return found_param
```

Comment:
i think it's preferable to directly `return trstats.get_param( .. )` rather than having to define a new variable